core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -615,6 +617,10 @@ oppia.directive('conversationSkin', [
           };
 
           $rootScope.$on('playerStateChange', function(evt, newStateName) {
+            var isLoggedIn = false;
+            UserService.getUserInfoAsync().then(function(userInfo) {
```

Comment:
This might be problematic, because it happens asynchronously. What if the async call takes a long time and the stuff below executes before this function call returns?

Reply:
Used `$scope.isLoggedIn` instead, I missed it earlier.